orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/CloudwatchMetricsService.java

```diff
@@ -90,12 +90,12 @@ public void incrementLogout(
             Optional<AccountInterventionStatus> accountInterventionStatus) {
         String accountInterventionStr = "unknown";
         if (accountInterventionStatus.isPresent()) {
-            if (accountInterventionStatus.get().blocked()) {
-                accountInterventionStr = "blocked";
-            }
             if (accountInterventionStatus.get().suspended()) {
                 accountInterventionStr = "suspended";
             }
+            if (accountInterventionStatus.get().blocked()) {
```

Comment:
Good spot! Would it be better to use a more descriptive string eg. `SUSPENDED_NO_ACTION`?

Reply:
I'm not sure what you mean?

Reply:
Will resolve this as it's not blocking approval of PR